### ISSUE TITLE
[gree] 8°C heating function for Gree Air Conditioner binding

### ISSUE THREAD

**Issue by antonmies** [openhab/openhab-addons#9438]
I use the 'New Gree Air Conditioner Binding' in OH 2.5.x and 3.0 M4 with GWH09QB-K6DNB8 that has in built 8°C heating feature.

There is a reference to the commands in https://github.com/tomikaa87/gree-remote/issues/1#issuecomment-430030212

Quote: "Just checked with an android packet capturing application and can confirm that "opt":"StHt","p":"1.0" is used to turn on and "opt":"StHt","p":"0.0" is used to turn off the 8°C heating mode."

I have also logged some packet data with Wireshark, but they seem to be encrypted and I don't know how to decrypt them to confirm the above commands.

**Comment by antonmies** [comment]
I was just testing different settings with the new version and noticed that the function differs from the Gree app:

In the APP: When the safety heating is set ON, target temperature value is set to 8degC and when the SafetyHeat function is set OFF, the target temperature value changes to whatever it was prior to setting the safety heat mode ON.

The Gree APP doesn't allow the safety heating enabled when UnitMode is NOT set to 4 / "HEAT". 
If safety heating is set to ON and mode is changed, the SafetyHeat is automatically set OFF and mode is changed to whatever it's set to and SafetyHeat can not be enabled before changing UnitMode back to 4.

The binding behavior: If the UnitMode is changed while the SafetyHeat is set to ON, the SafetyHeat indication is not updated any more so it is possible to have an indication that the SafetyHeat is toggled differently than it actually is.
Also, if SafetyHeat is enabled or disabled from the Gree APP, the status indication does not seem to change in OH.

**Comment by markus7017** [comment]
So it's just a problem to keep the safetyHeating channel in sync?
= Command is working, state update inconsistent?

**Comment by antonmies** [comment]
Yes, the command works (ON-OFF) but state update is not perfect as described.

**Comment by antonmies** [comment]
Came here to bump this as the feature itself seems to be missing from the binding despite it was functional earlier.

The 8C heating mode (StHt) should be added as it saves energy and and doesn't rely on outside control (like OH as thermostat control) to function.

**Comment by antonmies** [comment]
Thank you @Hilbrand for noticing this at #13799. I had a feeling this was a bit buried by looking at the dates.

Yes, it was a custom build by @markus7017 , who made it work and continued the original work of @jllcunha, but I guess it never made to the official binding - which I never installed before I upgraded to OH3 so I didn't really notice this before until recently.

Who are now the ones maintaining this binding?

**Comment by Hilbrand** [comment]
@antonmies, markus7017 is still registered as the codeowner in the [CODEOWNERS](https://github.com/openhab/openhab-addons/blob/6bdcd15d9963479ff2afe6dc5806dd5c90523c2e/CODEOWNERS#L118) file.

@markus7017 do you still have the code with the `safetyheat` channel? I couldn't find it in you repo.
